lib/chef/provider.rb

```diff
@@ -357,6 +357,16 @@ def initialize(resource, run_context)
         super(resource, run_context.create_child)
       end
 
+      def converge_action(action)
```

Comment:
Can you add a comment explaining why this is defined both in this module and on the provider class?